XMonad/Actions/GroupNavigation.hs

```diff
@@ -216,3 +220,22 @@ findM cond xs = findM' cond (viewl xs)
       if isMatch
         then return (Just x')
         else findM qry xs'
+
+
+-- $utilities
+-- #utilities#
```

Comment:
What is this `#utilities#` line for?

Reply:
I have never done haddock documentation, and followed an example. I assume it is for an HTML anchor but not sure; just followed the documentation of another `Actions.*` module.

Reply:
Ah, I learned something new today!  You're right, it is to make an HTML anchor: https://www.haskell.org/haddock/doc/html/ch03s08.html#idm140354810720016